web/client-ui/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM deephaven/node:local-build
 WORKDIR /usr/src/app
 
 # Most of the time, these versions are the same, except in cases where a patch only affects one of the packages
-ARG WEB_VERSION=0.25.0
-ARG GRID_VERSION=0.25.0
-ARG CHART_VERSION=0.25.0
+ARG WEB_VERSION=0.24.1-hierarchical-table.0
+ARG GRID_VERSION=0.24.1-hierarchical-table.1
+ARG CHART_VERSION=0.24.1-hierarchical-table.0
```

Comment:
Pull this out of your change - I'll publish v0.26.0 shortly and update the version in core in a separate ticket (linking to release notes etc).

Reply:
v0.26.0 has been published and added to core already

Reply:
I've removed these patches, so will just lean on upstream.